src/components/date-time-input/date-time-input.spec.ts

```diff
@@ -982,7 +982,7 @@ describe('Date Time Input component', () => {
     });
 
     it('fulfils min value constraint', () => {
-      spec.setProperties({ min: new Date(2026, 0, 1) });
+      spec.setProperties({ min: new Date(2026, 0, 1), value: today.native });
```

Comment:
Eh you might wanna just change the expect since it's supposed to pass with no value and required :)
This can start passing in two years time :D